README.md

```diff
@@ -15,3 +15,10 @@ The main differences between this provisioner and the standard hostpath provisio
 ## Deployment
 
 The provisioner is deployed as a daemonset, and instance of the provisioner is deployed to each of the worker nodes in the kubernetes cluster.  We then disable the use of leader election so that any provisioning request is issues to all of the provsioners in the cluster.  Each provisioner then evaluates the provision request based on the Node attribute by filtering out any requests that don't match the Node name for the provisioner pod.
+
+### Deployment in Open Shift
```

Comment:
OpenShift always (without a space between the words)

Reply:
Done